packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -46,10 +46,10 @@ export default function clearFormat(editor: Editor) {
                 setFontSize(editor, defaultFormat.fontSize);
             }
             if (defaultFormat.textColor) {
-                setTextColor(editor, defaultFormat.textColor);
+                setTextColor(editor, defaultFormat.textColor, defaultFormat.ogsc);

```

Comment:
I know ogsc represents the color for light mode, but why not just say that directly? e.g. defaultFormat.lightMode